wolfssl/wolfio.h

```diff
@@ -62,15 +62,12 @@
         #include <rtcs.h>
     #elif defined(FREESCALE_KSDK_MQX)
         #include <rtcs.h>
-    #elif defined(WOLFSSL_MDK_ARM) || defined(WOLFSSL_KEIL_TCP_NET)
-        #if !defined(WOLFSSL_MDK_ARM)
-            #include "cmsis_os.h"
-            #include "rl_net.h"
-        #else
-            #include <rtl.h>
-        #endif
+    #elif (defined(WOLFSSL_MDK_ARM) || defined(WOLFSSL_KEIL_TCP_NET))&& defined(WOLFSSL_CMSIS_RTOS)
```

Comment:
For readability please add space and make sure its 80 character limited. Example:

```
#elif (defined(WOLFSSL_MDK_ARM) || defined(WOLFSSL_KEIL_TCP_NET)) \
        && defined(WOLFSSL_CMSIS_RTOS)
```

Reply:
Removed WOLFSSL_CMSIS_RTOS